android/res/values-zh/strings.xml

```diff
@@ -431,7 +440,8 @@
 	<string name="editor_report_problem_other_title">一个不同的问题</string>
 	<string name="placepage_add_business_button">添加组织</string>
 	<string name="message_invalid_feature_position">对象无法设置在这里</string>
-	<string name="osm_presentation">所有地图数据均来自 OpenStreetMap 项目。它就像 Wikipedia 中的地图，您可以在其中添加或编辑地点，并且它们可供全球数百万用户使用。</string>
+	<!-- Text in About and OSM Login screens. %@ is replaced by a local, human readable date. -->
+	<string name="osm_presentation">地图数据来自截至 %s 的 OpenStreetMap。它就像地图的维基百科，您可以在其中为全世界所有用户添加或编辑地点。</string>
```

Comment:
https://github.com/LaoshuBaby/organicmaps/commit/54ea72dd381c26e92081e3a2822e7370abe9afa3 I don't have access to file so I make this commit according to https://github.com/organicmaps/organicmaps/blob/master/docs/TRANSLATIONS.md

Reply:
```
zh-Hans = 地图数据来自截至 %@ 的 OpenStreetMap。它就像地图的维基百科，您可以在其中添加或编辑可供全世界所有用户使用的地点。
```